README.md

```diff
@@ -1,5 +1,17 @@
 **DISCLAIMER: THIS IS A WORK IN PROGRESS**
 
-# BigchainDB RFC
+# BigchainDB Enhancement Proposals (BEPs)
 
-This is the BigchainDB RFC project.
+This is the BigchainDB Enhancement Proposal project. We collect specifications for APIs, protocols, and processes.
+
+The process to add or change an EIP is the following:
+- A specification is created and modified by pull requests according to [C4](./1).
```

Comment:
I wonder why you used the word "specification" here (and on the next lines) but elsewhere you were thorough in replacing all "specifications" with "BEPs".

Reply:
Thanks for pointing out, I forgot to change this file.